lib/ex_machina/ecto.ex

```diff
@@ -71,6 +71,7 @@ defmodule ExMachina.Ecto do
   def params_for(module, factory_name, attrs \\ %{}) do
     module.build(factory_name, attrs)
     |> drop_ecto_fields
+    |> drop_nil_values
```

Comment:
Yeah I think that was a good idea to change this. Maybe `drop_fields_with_nil_valudes`?